libraries/cli/include/cli/config_jsons/devnet/config.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
I think it might be more handy if we call these files devnet_genesis and devnet_config ? Otherwise when adjust these configs, the only way how do differenciate between testnet/maiinet/devnet/default will be file path